torch_em/segmentation.py

```diff
@@ -1,7 +1,7 @@
 import os
 from glob import glob
 
-import torch
+import torch.utils.data
```

Comment:
For me this is sufficient and this works:
```python
import torch
loader = torch.utils.data.DataLoader
```
Maybe depends on the torch version?

Reply:
oh, true, I was misled by PyCharm complaining. I think there is some lazy module loading involved for torch.utils. If you don't mind I would extend it for static type checks. 

utils magic in `torch.__init__.py`:
```python
...
from ._utils import _import_dotted_name
from ._utils_internal import get_file_path, prepare_multiprocessing_environment, \
    USE_RTLD_GLOBAL_WITH_LIBTORCH, USE_GLOBAL_DEPS
    ...
```

Reply:
But we use `torch` in several other places, e.g. https://github.com/constantinpape/torch-em/blob/main/torch_em/segmentation.py#L189.
So this change would break several things.

Reply:
I don't see this breaking anything..
```python
>>> import torch.utils.data
>>> torch.float32
torch.float32
```
I mean torch.__init__ has to execute when importing `torch.<anything>`, or am I missing some fundamental python quirks?

Reply:
Yeah, you're right this still works (though I think it's possible to write mean things that would make this not work). It's just conceptually weird to ask for a specific torch module when you use functionality from the top-level one.